### ISSUE TITLE
Not working with Flutter 3.10

### ISSUE THREAD

**Issue by YarostheLaunchpadder** [SachinGanesh/screenshot#137]
I upgraded to Flutter 3.10 and I'm getting the following error when building the app:

```gradle
../../../AppData/Local/Pub/Cache/hosted/pub.dev/screenshot-1.3.0/lib/screenshot.dart:160:7: Error: No named parameter with the name 'window'.
      window: ui.window,

../../flutter/packages/flutter/lib/src/rendering/view.dart:68:3: Context: Found this candidate, but the arguments don't match.
  RenderView({
  ^^^^^^^^^^
```

This is because `window` isn't a valid argument of `RenderView` anymore: 

<img src="https://github.com/SachinGanesh/screenshot/assets/54041533/bd4fbeca-565a-4e97-beb1-8a40ff77099f" width="600px" />

**Comment by h3x4d3c1m4l** [comment]
This should be fixed by PR #132. It has yet to be released to pub.dev.

**Comment by mariopepe** [comment]
Having this issue as well!

**Comment by Subrataporwal** [comment]
> This should be fixed by PR #132. It has yet to be released to pub.dev.

Please Update us on release

**Comment by tmaihoff** [comment]
Thanks for the fast fix. Waiting for pub release as well ☕

**Comment by luisrodas-digital** [comment]
Yes please, Waiting for release :D

**Comment by luisrodas-digital** [comment]
dart pub publish 😄

**Comment by marcellocamara** [comment]
Unfortunately the owner is taking too long to fix it
Setting this at your `pubspec.yaml` will work:

```
  screenshot:
    git:
      url: https://github.com/SachinGanesh/screenshot.git
      ref: master
```

**Comment by h3x4d3c1m4l** [comment]
It seems to be published: https://pub.dev/packages/screenshot

**Comment by ritheshSalyan** [comment]
Published version `2.0.0` with support for flutter `3.10`

**Comment by Subrataporwal** [comment]
Yes Working Perfectly Fine
